src/main/java/com/bamdoliro/bgcr/domain/user/BgcrUser.java

```diff
@@ -0,0 +1,57 @@
+package com.bamdoliro.bgcr.domain.user;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Entity
+@NoArgsConstructor
```

Comment:
이게... `@Data`가 없다고 그 안에 있는 다른 것들까지 필요가 없는 건 아님
```suggestion
@Getter
@Entity
@Table(name = "tbl_user")
@NoArgsConstructor(access = AccessLevel.PROTECTED)
```